config/betterquesting/DefaultQuests.json

```diff
@@ -156272,7 +156272,7 @@
           "lockedProgress:1": 0,
           "autoClaim:1": 0,
           "isSilent:1": 0,
-          "desc:8": "Now that you have some tech, you\u0027ve discovered a way to prospect for ores. This tool only has a chance at finding the ores in a chunk. If you are clever, you can use it to locate ores.\n\nUnfortunately, the tech to make it repairable or rechargeable doesn\u0027t work, so you\u0027ll have to make a new one each time it breaks. It does work better at higher tiers and lasts longer with better materials.\n\nAnd since it doesn\u0027t make a convenient book like the seismic prospector, you\u0027ll have to keep track on your own where the ores are found.\n\nIt can be hard to choose a good material for the Basic Prospector. Here are some good choices for you, the size searched, and their durability:\n\nArdite 5x5 - 4500\nDamascus 3x3 - 5600\nVanadiumsteel 3x3 - 8400\nVibrant Alloy 5x5 - 17700\nShadow 5x5 - 36000\n\nEach chunk successfully scanned costs 100 durability. Each failed chunk costs 25. The average for MV is ~55 per chunk when scanning a large area.\n\nGood luck.\n\n§3BTW, in case you didn\u0027t get it, this thing is terrible."
+          "desc:8": "Now that you have some tech, you\u0027ve discovered a way to prospect for ores. This tool only has a chance at finding the ores in a chunk. If you are clever, you can use it to locate ores.\n\nUnfortunately, the tech to make it repairable or rechargeable doesn\u0027t work, so you\u0027ll have to make a new one each time it breaks. It does work better at higher tiers and lasts longer with better materials.\n\nAnd since it doesn\u0027t make a convenient book like the seismic prospector, you\u0027ll have to keep track on your own where the ores are found.\n\nIt can be hard to choose a good material for the Basic Prospector. Here are some good choices for you in their respective tiers, the size searched, and the average amount of veins they can scan:\n\nLV Vanadiumsteel 3x3 - 50\nMV Vanadiumsteel 3x3 - 213\nHV Vibrant Alloy 5x5 - 299\nEV Shadow Metal 7x7 - 1080\nIV Oriharukon 7x7 - 2649\n\nEach chunk successfully scanned costs 100 durability. Each failed chunk costs 25. The average for MV is ~55 per chunk when scanning a large area. As you can see, the prospectors become better and better with each tier, so craft one whenever you think you need it.\n\nGood luck."
```

Comment:
t make a convenient book like the seismic prospector, you\u0027ll have to keep track on your own where the ores are found

remove this - visual prospector solves everything imo

Reply:
Changed.